.github/dependabot.yml

```diff
@@ -76,6 +76,8 @@ updates:
 
     schedule:
       interval: monthly
+    cooldown:
+      default-days: 4
```

Comment:
What's the thinking behind 4 days?

Reply:
Sure. I assumed GitHub actions are safer as we’re only using the ones authored by GitHub. But I guess a cooldown is probably fine there too?

Reply:
Updated.